src/commonMain/kotlin/com/notkamui/keval/KevalBuilder.kt

```diff
@@ -172,8 +175,40 @@ class KevalBuilder internal constructor(
             "ceil" to KevalFunction(1) { ceil(it[0]) },
             "floor" to KevalFunction(1) { floor(it[0]) },
             "round" to KevalFunction(1) { round(it[0]) },
+            "trunc" to KevalFunction(1) { it[0].toInt().toDouble() },
+            "min" to KevalFunction(null) { it.min() },
+            "max" to KevalFunction(null) { it.max() },
+            "sum" to KevalFunction(null) { it.sum() },
+            "avg" to KevalFunction(null) { it.average() },
+            "median" to KevalFunction(null) { it.sorted()[it.size / 2] },
+            "percentile" to KevalFunction(null) {
+                if (it.size > 1) throw KevalInvalidArgumentException("percentile requires at least 2 values")
+                val perc = it[0]
+                if (perc in 0.0..100.0) throw KevalInvalidArgumentException("percentile must be between 0 and 100")
```

Comment:
I suppose you inverted the comparison

```suggestion
                if (perc !in 0.0..100.0) throw KevalInvalidArgumentException("percentile must be between 0 and 100")
```